package.json

```diff
@@ -39,6 +39,7 @@
     "postcss": "^8.2.1",
     "serve": "^11.3.2",
     "simple-git": "^2.24.0",
+    "strip-comments": "^2.0.1",
```

Comment:
Small nit: Shouldn't this be a normal dependency? It's used to power a HBS helper.

Reply:
this is only used for tests, the real dependency is in static/package.json

Reply:
actually, we don't need this dep at all. javascript is pretty smart about how it parses out comments